src/marten/http/uploaded_file.cr

```diff
@@ -7,13 +7,25 @@ module Marten
       # Returns the `File` object associated with the corresponding temporary file.
       getter io
 
-      def initialize(@part : ::HTTP::FormData::Part)
+      def initialize(@part : ::HTTP::FormData::Part, @ct : String = "text/plain")
```

Comment:
The issue with this is that `@ct` will always be set to text/plain even if the file corresponds to something else. I guess at least this should be set based on the filename or the content type in the multipart segment (we certainly don't want to infer this from the file blob).

Reply:
fixed in 72c309f5c20f522e6a440e582a97346eec6fb888